scene/gui/item_list.cpp

```diff
@@ -1475,6 +1477,9 @@ void ItemList::force_update_list_size() {
 
 			float page = MAX(0, size.height - theme_cache.panel_style->get_minimum_size().height);
 			float max = MAX(page, ofs.y + max_h);
+			if (auto_width) {
+				auto_width_value = ofs.x + max_w + theme_cache.panel_style->get_minimum_size().width + 6; // Last letters of a word are missing without 6 additional pixels.
```

Comment:
The hard-coded value may vary depending on font size, scaling or other factors. It's better to find where the difference is coming from and fix it properly.